e2e/pages/appointments-page.ts

```diff
@@ -0,0 +1,11 @@
+import { type Page } from '@playwright/test';
+
+export class AppointmentsPage {
+  constructor(readonly page: Page) {}
+
+  readonly appointmentsTable = () => this.page.getByRole('table', { name: /appointments/i });
```

Comment:
```suggestion
  readonly appointmentsTable = () => this.page.getByRole('table', { name: /appointments/i });
```
I think it's best to Add a getByTestId to the table locator

Reply:
It's better to avoid the use of ids, classes as much as possible. https://github.com/openmrs/openmrs-esm-patient-chart/pull/1664#discussion_r1494559226